packages/@okta/vuepress-site/docs/api/resources/oidc/index.md

```diff
@@ -516,6 +516,8 @@ These keys can be used to locally validate JWTs returned by Okta. Standard open-
 | :---------- | :---------------------------- | :----------- | :--------- | :--------- | :------ |
 | client_id   | Your app's client ID.         | Query        | String     | FALSE      | null    |
 
+>Note that the request parameter `client_id` is only applicable to Okta Org Authorization Server. See [Composing Your Base URL](/docs/api/resources/oidc/#composing-your-base-url) for more information.
```

Comment:
What is the `more information` the `Composing Your Base URL` will provide?

Also in the second paragraph of `Key Rotation`, we should mention that client can only opt-out for Okta  Org AS only.

Reply:
I'm guessing that he links to it because it explains the difference between Org and Custom Authorization Servers.